x-pack/plugins/security_solution/public/common/components/page/use_refetch_by_session.tsx

```diff
@@ -63,5 +61,9 @@ export const useRefetchByRestartingSession = ({
     );
   }, [dispatch, queryId, selectedInspectIndex, skip]);
 
-  return { searchSessionId, refetchByRestartingSession };
+  return {
+    session,
+    searchSessionId: existingSearchSessionId,
```

Comment:
do we need to be returning the `searchSessionId`? I don't see it being used anywhere